src/main/java/com/FoodyBuddy/Dao/ApartmentDAO.java

```diff
@@ -0,0 +1,12 @@
+package com.FoodyBuddy.DAO;
+
+import java.util.List;
+import com.FoodyBuddy.Model.Apartment;
+
+public interface ApartmentDAO {
+	public List<Apartment> getAllApartments();
```

Comment:
Don't over-explicit the names in all the DAO and models

instead if  getAllAppartmenets() make  getAll()